vpr/src/base/read_place.h

```diff
@@ -17,8 +17,15 @@ void read_place(
  */
 void read_constraints(const char* constraints_file);
 
+/**
+ * This function prints out a place file.
+ * @param is_place_file: defaults to true. If false, does not print file header; this is useful if
+ *                       the output will be used as a constraints file. If is_place_file is false,
+ *                       net_file and net_id parameters are not used and can be set to nullptr.
```

Comment:
Should also explain that if is_place_file is false, a partial placement can be printed (only placed block will have their placement output).